libraries/botbuilder/etc/botbuilder.api.md

```diff
@@ -366,6 +369,8 @@ export class TeamsActivityHandler extends ActivityHandler {
     protected dispatchMessageUpdateActivity(context: TurnContext): Promise<void>;
     protected handleTeamsAnonymousAppBasedLinkQuery(_context: TurnContext, _query: AppBasedLinkQuery): Promise<MessagingExtensionResponse>;
     protected handleTeamsAppBasedLinkQuery(_context: TurnContext, _query: AppBasedLinkQuery): Promise<MessagingExtensionResponse>;
+    protected handleTeamsBotConfigFetch(_context: TurnContext, _configData: any): Promise<ConfigResponse<BotConfigAuth>>;
```

Comment:
Could you explain why fetch and submit (the line below) are extending different types, I remember the logic under fetch and submit should be the same: they both have the opportunity to get auth or task response . Please correct me if I misunderstood here.

Reply:
For all other Bot Invoke. "Bot" was not used in the method name. Do you think it will be better to remove "Bot" in both 
`handleTeamsBotConfigFetch `and `handleTeamsBotConfigSubmit `so that it align with the name convention.